src/Makefile.am

```diff
@@ -651,6 +653,7 @@ libbitcoin_common_a_SOURCES = \
   policy/feerate.cpp \
   policy/policy.cpp \
   protocol.cpp \
+  proxy.cpp \
```

Comment:
Any reason to put this in common? I'd expect it can live next to conman, as the cli or wallet shouldn't be using this, no?

Might be better to put into `node/proxy.cpp`?